app/views/tests/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="form-horizontal">
+	<div class="row">
+		<div class="panel-body">
+			 
+		
+			
+					<%= form_for Question.new  do |f| %>
+			<% Question.all.each do |que| %>
+			<%=que.question%><br>
+			<%[que.option1,que.option2,que.option3,que.option4].each do |student|%>
```

Comment:
<%[que.option1,que.option2,que.option3,que.option4].each do |student|%>
why you have defined  student as parameter name, change it to 'option'.

<%[que.option1,que.option2,que.option3,que.option4].each do |student|%>
no need to create an array of it. 
you can just put it like 
<%= que.option1>
<%= que.option2>
<%= que.option3>
<%= que.option4>